crowdfunding/users/serializers.py

```diff
@@ -0,0 +1,47 @@
+from rest_framework import serializers, validators
+from .models import CustomUser
+
+from projects.serializers import CommentSerializer, ProjectSerializer, PledgeSerializer
+class CustomUserSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = CustomUser
+        fields = ('id', 'username', 'email', 'is_active', 'password','bio','avatar')
+        # fields = ('id', 'username', 'email', 'is_active', 'password','bio','avatar','comments','projects','pledges')
+
+        extra_kwargs = {
+            'email':{
+                'validators':
+                [validators.UniqueValidator(queryset=CustomUser.objects.all())],
+                'allow_blank':False,
+                'required':True},
```

Comment:
oh very interesting, i had just assumed that django model serializer would do this for you automatically? was that not the case?? 
very impressive validation step eitherway